odoo/addons/openupgrade_records/lib/compare.py

```diff
@@ -50,7 +55,7 @@ def compare_records(dict_old, dict_new, fields):
             if module_map(dict_old['module']) != dict_new['module']:
                 return False
         elif field == 'model':
-            if model_map(dict_old['model']) != dict_new['model']:
+            if model_rename_map(dict_old['model']) != dict_new['model']:
```

Comment:
Why are you changing here the called method?

Reply:
Well, because this one should still do the same as before. Another option should be not changing this one, but changing the other ones but it would imply more line diff.

Reply:
Something doesn't fit me or I don't have enough information, because if you change this, why changing also the method `model_map` for fallbacking to `merged_models` mapping?

Reply:
Because in the other cases where this method is called, you want to include the merged_models thing. It's only in this line where you don't want the merged_models thing included.

Reply:
OK, now go it, thanks.